project/Settings.scala

```diff
@@ -1,19 +1,24 @@
 import com.typesafe.sbt.SbtScalariform
 import com.typesafe.sbt.pgp.PgpKeys
 import sbtrelease.ReleasePlugin.autoImport._
-import scoverage.ScoverageSbtPlugin._
 import sbt.Keys._
 import sbt._
 import xerial.sbt.Sonatype
 
 object Settings {
 
+  val scala_2_11 = "2.11.8"
+  val scala_2_12 = "2.12.1"
+
   val alsoOnTest = "compile->compile;test->test"
 
   // settings for ALL modules, including parent
-  val core = Seq(
+  val common = Seq(
     organization := "org.virtuslab",
-    scalaVersion := "2.11.8",
+
+    parallelExecution in Test := false,
```

Comment:
I just moved the line - it was original here: https://github.com/VirtusLab/unicorn/pull/85/files/e5dd68d039362f30fba476e1b673ba786217c799#diff-6c323177457b467c4590f2132ba7f87aL50

I am guessing that it's because we create / drop same table in every test case.